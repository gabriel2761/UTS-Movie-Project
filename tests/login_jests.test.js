import  { HashRouter as Router, Switch, Route, Link } from 'react-router-dom';
import React from 'react';
import LoginPage from '../public/components/login.js';
import RegisterPage from '../public/components/register.js';
import renderer from 'react-test-renderer';

describe('Login Page renders correctly', () => {
	it('Renders correctly', () => {
		const rendered = renderer.create(<Router><LoginPage /></Router>);
		expect(rendered.toJSON()).toMatchSnapshot();
	});
});

describe('Register Page renders correctly', () => {
	it('Renders correctly', () => {
		const rendered = renderer.create(<Router><RegisterPage /></Router>);
		expect(rendered.toJSON()).toMatchSnapshot();
	});
});
