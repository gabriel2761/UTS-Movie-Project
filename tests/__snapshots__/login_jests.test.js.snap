// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creating a booking renders correctly Renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      placeholder="Username"
      type="text"
    />
    <br />
    <input
      placeholder="Password"
      type="text"
    />
    <br />
    <button
      type="submit"
    >
      Login
    </button>
  </form>
  <a
    href="#/register"
    onClick={[Function]}
  >
    Don't have an account yet? Sign Up Here!
  </a>
</div>
`;

exports[`Login Page renders correctly Renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      placeholder="Username"
      type="text"
    />
    <br />
    <input
      placeholder="Password"
      type="text"
    />
    <br />
    <button
      type="submit"
    >
      Login
    </button>
  </form>
  <a
    href="#/register"
    onClick={[Function]}
  >
    Don't have an account yet? Sign Up Here!
  </a>
</div>
`;

exports[`Register Page renders correctly Renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      placeholder="Username"
      type="text"
    />
    <br />
    <input
      placeholder="Password"
      type="text"
    />
    <br />
    <button
      type="submit"
    >
      Signup
    </button>
  </form>
  <a
    href="#/login"
    onClick={[Function]}
  >
    Already have an account? Login Here!
  </a>
</div>
`;
